docusaurus.config.js

```diff
@@ -23,23 +23,14 @@ module.exports = {
   // do not delete the following 'noIndex' line as it is modified for production
   noIndex: true,
   plugins: [
-    //        ["@edno/docusaurus2-graphql-doc-generator",
-    //          {
-    //            schema: "http://localhost:8080/tasklist/graphql",
-    //            rootPath: "./docs/", // docs will be generated under (rootPath/baseURL)
-    //            baseURL: "apis-tools/tasklist-api",
-    //            linkRoot: "/docs/",
-    //            loaders: {
-    //              UrlLoader: "@graphql-tools/url-loader"
-    //            }
-    //          },
-    //        ],
```

Comment:
I don't know what this is or why were holding on to it, but I've been meaning to delete this for a long time 😅